src/substrate/substrate.py

```diff
@@ -16,7 +16,7 @@
 	'ospray_studio': OSPRayStudio,
 	'tapestry': Tapestry,
 	'vci': VCI,
-	'DCHM': DCHM 
+	'dchm': DCHM 
```

Comment:
There is a trailing whitespace on this line. Also, can you move this one to the top to keep things alphabetical?